Improve translations.

salvoscala opened the pull request:
A manager should be able to translate:
- The Global customer message;
- Global site settings;
- T&C;

salvoscala committed:
Make the customer message translatable.

salvoscala committed:
Code style.

salvoscala committed:
Make T&C translatables.

salvoscala committed:
Add descriptions to certain fields

Commit:
Make T&C translatables

salvoscala committed:
Translatable popup in conversations.

salvoscala committed:
Remove space

Commit:
Make ‘Custom Offline Payment’ settings translatable

Commit:
Workaround to enable or disable rule 'commerce_payment_custom'

salvoscala committed:
Customer should not be able to remove Mandatory options. (#88)

salvoscala committed:
Change text format.

salvoscala committed:
Make the checkout completition message translatable.

Commit:
Add PHP 7.1 compatibility

Commit:
Make the checkout completition message translatable

salvoscala committed:
Make the message popup Close button translatable.

salvoscala committed:
Code style.

acrollet commented on modules/roomify/roomify_conversations/roomify_conversations.module:
```diff
@@ -1675,7 +1675,7 @@ function roomify_conversations_quick_reply_form($form, &$form_state) {
 
   $form['property_title'] = array(
-    '#markup' => '<h2>Conversation with ' . $guest->name . '</h2>',
+    '#markup' => t('<h2>Conversation with @guest</h2>', array('@guest' => $guest->name)),
```
shouldn't the h2 tags be outside of the call to t()?

acrollet commented on modules/roomify/roomify_payment_methods/roomify_payment_methods.module:
```diff
@@ -263,51 +265,55 @@ function roomify_payment_methods_settings($form, &$form_state) {
 
 /**
- *
+ * Validate function for the roomify_payment_methods_settings form.
```
nbd but should be 'Validation function' :)

acrollet reviewed (approved):
passes CR after minor changes

salvoscala committed:
Minor changes.

salvoscala committed:
Booking information storage improvements. (#84)

* Create new orders for booking.com and ical bookings.

* New Customer profile on new orders

* Same booking edit form for all types

* Add booking_line_item_reference field to all booking types  and show customer profile info with edit link

* Add new order for all existing booking_com or ical bookings.

* Show order summary with view link

* Fix tests

* Fix tests

* Move code to the right hook

* Add module 'References dialog'

* Don’t show “Documentation” button on references dialog

* Remove Address fields from booking types

* Migrate all existing guest data to a new customer profile

* Fix creation of 'booking_telephone' field for customer profile

* Associate customer profile on new booking

* Prevent errors if custom profile is not setted

* Change updates order

* Update views on the dashboard to show guest informations.

* Further improve button text based on 'Instant Booking' status (#33)

* Further improve button text based on 'Instant Booking' status.

* Reduce room type name font size.

* Add commerce addressbook module (#24)

* Add commerce addressbook module and theme it.

Recreated the homepage blogpost view to remove pager.

* Change button text and color.

* Update smtp module to 1.6

* Theming improvements. (#36)

* Improve base themes. (#38)

* Add the Roomify Integrations module

* Base the theme colors tab access on a permission

* Improve properties fields language fallback behavior

* Make landing page titles translatable

* Do not re-arrange filesystem when packaging RfA

* Remove hook_mail invocation from roomify.profile (#51)

* Update channel connector module to 1.11

* Show the correct price if a per-person rate is enabled

* Metatag support for Roomify Properties and Bat Entities (#55)

* Update BAT and Property

* Enable metatags on Type and Property

* Mysql 5.7 compatibility and fix tests (#70)

* Update ‘Roomify iCal Channel’ to v. 1.3

* Fix test dates

* Update modules (#47)

* Update modules

* Remove patches on editablefields

* Further improve button text based on 'Instant Booking' status.

* Improve test

* Patch token

* Add comment.

* Improve travel theme (#76)

* Remove all @extend from travel theme

* Remove all @extend from travel theme

* Remove all @extend from travel theme

* General theme improvements

* Change colors of placeholder and button on the availability search form in the homepage

* Remove the editablefields module

* Update  Roomify travel theme header

* Use translated titles on activities, posts and locations

* Update single property page. (#58)

* Follow-on to Image pushes map tooltip off of screen #35 (#74)

* Image pushes map tooltip off of screen and small theme fixes

* Improve booking page on the wide theme.

* Improve booking page on the wide theme.

* Improve theme of tooltips.

* Remove all @extend from travel theme

* Improve CSS

* Minor css fixes.

* Change language row's color in the roomify bootstrap themes

* Further improve button text based on 'Instant Booking' status (#33)

* Further improve button text based on 'Instant Booking' status.

* Reduce room type name font size.

* Add commerce addressbook module (#24)

* Add commerce addressbook module and theme it.

Recreated the homepage blogpost view to remove pager.

* Change button text and color.

* Update smtp module to 1.6

* Theming improvements. (#36)

* Improve base themes. (#38)

* Add the Roomify Integrations module

* Base the theme colors tab access on a permission

* Improve properties fields language fallback behavior

* Make landing page titles translatable

* Do not re-arrange filesystem when packaging RfA

* Remove hook_mail invocation from roomify.profile (#51)

* Update channel connector module to 1.11

* Show the correct price if a per-person rate is enabled

* Metatag support for Roomify Properties and Bat Entities (#55)

* Update BAT and Property

* Enable metatags on Type and Property

* Mysql 5.7 compatibility and fix tests (#70)

* Update ‘Roomify iCal Channel’ to v. 1.3

* Fix test dates

* Update modules (#47)

* Update modules

* Remove patches on editablefields

* Further improve button text based on 'Instant Booking' status.

* Improve test

* Patch token

* Add comment.

* Add offers option for "Always override price" (#81)

* Fix wrong departure date on enquiries. (#83)

* Only enable required modules when an integration is enabled (#49)

* Only enable required modules when an integration is enabled.

* Remove empty hook.

* Remove the editablefields module

* Update  Roomify travel theme header

* Use translated titles on activities, posts and locations

* Update single property page. (#58)

* Follow-on to Image pushes map tooltip off of screen #35 (#74)

* Image pushes map tooltip off of screen and small theme fixes

* Improve booking page on the wide theme.

* Improve booking page on the wide theme.

* Improve theme of tooltips.

* Further improve button text based on 'Instant Booking' status (#33)

* Further improve button text based on 'Instant Booking' status.

* Reduce room type name font size.

* Add commerce addressbook module (#24)

* Add commerce addressbook module and theme it.

Recreated the homepage blogpost view to remove pager.

* Change button text and color.

* Update smtp module to 1.6

* Theming improvements. (#36)

* Improve base themes. (#38)

* Add the Roomify Integrations module

* Base the theme colors tab access on a permission

* Improve properties fields language fallback behavior

* Make landing page titles translatable

* Do not re-arrange filesystem when packaging RfA

* Remove hook_mail invocation from roomify.profile (#51)

* Update channel connector module to 1.11

* Show the correct price if a per-person rate is enabled

* Metatag support for Roomify Properties and Bat Entities (#55)

* Update BAT and Property

* Enable metatags on Type and Property

* Mysql 5.7 compatibility and fix tests (#70)

* Update ‘Roomify iCal Channel’ to v. 1.3

* Fix test dates

* Update modules (#47)

* Update modules

* Remove patches on editablefields

* Further improve button text based on 'Instant Booking' status.

* Improve test

* Patch token

* Add comment.

* Add a roomify_integrations db table

* Improve travel theme (#76)

* Remove all @extend from travel theme

* Remove all @extend from travel theme

* Remove all @extend from travel theme

* General theme improvements

* Change colors of placeholder and button on the availability search form in the homepage

* Remove the editablefields module

* Update  Roomify travel theme header

* Use translated titles on activities, posts and locations

* Update single property page. (#58)

* Follow-on to Image pushes map tooltip off of screen #35 (#74)

* Image pushes map tooltip off of screen and small theme fixes

* Improve booking page on the wide theme.

* Improve booking page on the wide theme.

* Improve theme of tooltips.

* Remove all @extend from travel theme

* Improve CSS

* Minor css fixes.

* Change language row's color in the roomify bootstrap themes

* Further improve button text based on 'Instant Booking' status (#33)

* Further improve button text based on 'Instant Booking' status.

* Reduce room type name font size.

* Add commerce addressbook module (#24)

* Add commerce addressbook module and theme it.

Recreated the homepage blogpost view to remove pager.

* Change button text and color.

* Update smtp module to 1.6

* Theming improvements. (#36)

* Improve base themes. (#38)

* Add the Roomify Integrations module

* Base the theme colors tab access on a permission

* Improve properties fields language fallback behavior

* Make landing page titles translatable

* Do not re-arrange filesystem when packaging RfA

* Remove hook_mail invocation from roomify.profile (#51)

* Update channel connector module to 1.11

* Show the correct price if a per-person rate is enabled

* Metatag support for Roomify Properties and Bat Entities (#55)

* Update BAT and Property

* Enable metatags on Type and Property

* Mysql 5.7 compatibility and fix tests (#70)

* Update ‘Roomify iCal Channel’ to v. 1.3

* Fix test dates

* Update modules (#47)

* Update modules

* Remove patches on editablefields

* Further improve button text based on 'Instant Booking' status.

* Improve test

* Patch token

* Add comment.

* Fix typo.

* Filter bookings in the sidebar by date. (#78)

* Add images to the quick setup page. (#77)

* Show logo after upload. (#68)

* Minor updates to comments/formatting to improve code style

* Add some featured properties when a multiproperty example content is enabled. (#48)

* Add some featured properties when a multiproperty example content is enabled.

* Code style.

* Let the user choose to leave homepage nodes.

* Add check.

* Fix tests.

* Add descriptions to certain fields

* Customer should not be able to remove Mandatory options. (#88)

* Add guest informations to views.

* A roomify manager should be able to add, view and edit customer profiles.

salvoscala committed:
Merge branch 'master' into issue-87
